sky/skylet/providers/kubernetes/utils.py

```diff
@@ -66,3 +66,18 @@ def get_cluster_status(cluster_name: str,
             cluster_status.append(status_lib.ClusterStatus.INIT)
     # If pods are not found, we don't add them to the return list
     return cluster_status
+
+
+def get_current_kube_config_context() -> Union[str, None]:
```

Comment:
nit: we use `Optional[str]` instead of `Union[str, None]` in our codebase

Reply:
👍 Updated in https://github.com/skypilot-org/skypilot/pull/2253/commits/ef01c15ac697c0a2618e70960e843c347970db00